libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -1,7 +1,7 @@
 import { getSheet } from '../../../../tools/utils/utils.js';
 import { LitElement, html } from '../../../deps/lit-all.min.js';
 import { Task } from '../../../deps/lit-task.min.js';
-import login from '../../../tools/sharepoint/login.js';
+// import login from '../../../tools/sharepoint/login.js';
```

Comment:
I assume we'll shortly need the code that's commented out and that's why you haven't removed it.

Reply:
Yes, that is correct. I commented it out to avoid the eslint check failing.